tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -130,8 +131,10 @@ Feature: Duplicate product from Back Office (BO).
       | en-US  | Simple & nice sunglasses   |
       | fr-FR  | lunettes simples et belles |
     And product copy_of_product1 should be assigned to following categories:
-      | categories       | [home, men, clothes] |
-      | default category | clothes              |
+      | id reference | name[en-US] | name[fr-FR] | is default |
+      | home         | Home        | Home        | false      |
```

Comment:
What worries me here is that the french values are not translated? Is it an issue with the fixtures? The install process? Or does it highlight a bug somewhere?

Reply:
Ok I just checked, indeed the category's french content is still not available in the test fixtures data We'll have to fix this someday But for now, can you create at least one category in this behat background that has french content so that we can truly check that the two languages are correctly fetched?
Not all of them of course but just one would be nice

Reply:
I would, but by default our behats supports creating category only in default language, I really don't want to mess with the PR scope again.
Maybe next time? :smile:

Reply:
added todo in behats